Gemfile

```diff
@@ -18,6 +18,7 @@ gem "plek"
 gem "redis"
 gem "rubocop-govuk"
 gem "sassc-rails"
+gem "sprockets-rails"
```

Comment:
This is a good spot! I'm guessing some tests broke when this was removed?

Reply:
To be honest, I didn't re-run the tests - but I'd guess they would have failed. All I did was check that we're still using it and add it back in as per the upgrade doc.